src/bundle/upload.ts

```diff
@@ -34,6 +34,7 @@ interface Options extends OptionsBase {
   ignoreMetadataCheck?: boolean
   timeout?: number
   multipart?: boolean
+  disableChecksumVerification?: boolean;
```

Comment:
You didn't test this either, the options are not added to the index file.
Honestly, I feel you make me lose my time here

Reply:
Testing it right now!